server/config.ts

```diff
@@ -74,6 +74,14 @@ export default {
       },
       agent: new AgentConfig(Number(get('NOMIS_PRISONER_TIMEOUT_RESPONSE', 60000))),
     },
+    mapping: {
+      url: get('MAPPING_API_URL', 'http://localhost:8103', requiredInProduction) as string,
+      timeout: {
```

Comment:
I think you might need this in the helm files else won't it stop the app from starting in dev?